code/modules/client/preferences_savefile.dm

```diff
@@ -174,46 +174,47 @@
+	show_typing = sanitize_integer(show_typing, FALSE, TRUE, initial(show_typing))
+	ghost_hud = sanitize_integer(ghost_hud, NONE, MAX_BITFLAG, initial(ghost_hud))
+	windowflashing = sanitize_integer(windowflashing, FALSE, TRUE, initial(windowflashing))
+	auto_fit_viewport = sanitize_integer(auto_fit_viewport, FALSE, TRUE, initial(auto_fit_viewport))
 	widescreenpref = sanitize_integer(widescreenpref, FALSE, TRUE, initial(widescreenpref))
 	pixel_size = sanitize_float(pixel_size, PIXEL_SCALING_AUTO, PIXEL_SCALING_3X, 0.5, initial(pixel_size))
-	scaling_method  = sanitize_text(scaling_method, initial(scaling_method))
-	ghost_vision	= sanitize_integer(ghost_vision, FALSE, TRUE, initial(ghost_vision))
-	ghost_orbit		= sanitize_inlist(ghost_orbit, GLOB.ghost_orbits, initial(ghost_orbit))
-	ghost_form		= sanitize_inlist_assoc(ghost_form, GLOB.ghost_forms, initial(ghost_form))
-	ghost_others	= sanitize_inlist(ghost_others, GLOB.ghost_others_options, initial(ghost_others))
-	clientfps		= sanitize_integer(clientfps, 0, 240, initial(clientfps))
+	scaling_method = sanitize_text(scaling_method, initial(scaling_method))
+	ghost_vision = sanitize_integer(ghost_vision, FALSE, TRUE, initial(ghost_vision))
+	ghost_orbit = sanitize_inlist(ghost_orbit, GLOB.ghost_orbits, initial(ghost_orbit))
+	ghost_form = sanitize_inlist_assoc(ghost_form, GLOB.ghost_forms, initial(ghost_form))
+	ghost_others = sanitize_inlist(ghost_others, GLOB.ghost_others_options, initial(ghost_others))
+	clientfps = sanitize_integer(clientfps, 0, 240, initial(clientfps))
 	parallax = sanitize_integer(parallax, PARALLAX_INSANE, PARALLAX_DISABLE, null)
-	tooltips		= sanitize_integer(tooltips, FALSE, TRUE, initial(tooltips))
+	tooltips = sanitize_integer(tooltips, FALSE, TRUE, initial(tooltips))
 
-	key_bindings 	= sanitize_islist(key_bindings, list())
-	custom_emotes   = sanitize_is_full_emote_list(custom_emotes)
-	chem_macros 	= sanitize_islist(chem_macros, list())
+	key_bindings = sanitize_islist(key_bindings, list())
+	custom_emotes = sanitize_is_full_emote_list(custom_emotes)
+	chem_macros = sanitize_islist(chem_macros, list())
+	quick_equip = sanitize_islist(quick_equip, QUICK_EQUIP_ORDER, MAX_QUICK_EQUIP_SLOTS)
 
-	mute_self_combat_messages	= sanitize_integer(mute_self_combat_messages, FALSE, TRUE, initial(mute_self_combat_messages))
-	mute_others_combat_messages	= sanitize_integer(mute_others_combat_messages, FALSE, TRUE, initial(mute_others_combat_messages))
-	mute_xeno_health_alert_messages	= sanitize_integer(mute_xeno_health_alert_messages, FALSE, TRUE, initial(mute_xeno_health_alert_messages))
+	mute_self_combat_messages = sanitize_integer(mute_self_combat_messages, FALSE, TRUE, initial(mute_self_combat_messages))
+	mute_others_combat_messages = sanitize_integer(mute_others_combat_messages, FALSE, TRUE, initial(mute_others_combat_messages))
+	mute_xeno_health_alert_messages = sanitize_integer(mute_xeno_health_alert_messages, FALSE, TRUE, initial(mute_xeno_health_alert_messages))
```

Comment:
Please atomize stuff like this in future it just makes it harder to review

Reply:
Will do, I just got over ambitious when I was looking at inventory code when the idea popped in my head.